core/web/keys_controller.go

```diff
@@ -100,6 +133,20 @@ func (kc *keysController[K, R]) Import(c *gin.Context) {
 		return
 	}
 
+	type_, err := keystore.GetFieldNameForKey(key)
```

Comment:
I think we can simplify a bit by doing this once in the constructor: https://github.com/smartcontractkit/chainlink/pull/7514
WDYT?